### DISCUSSION TITLE
Is it possible to freeze mermaid images so that they don't re-render eah time?

### DISCUSSION THREAD

**Discussion by jasonacollins** [quarto-dev/quarto-cli#7304] [Q&A]
### Description

My quarto blog has 'freeze: auto' as an execution option (set in the _quarto.yml file), with code only re-rendered during a global re-render in case of change.

The 'auto' option does not seem to apply to mermaid code blocks, which re-render every time.

Is it possible to freeze mermaid code blocks? If so, how? (If not, it would be a valuable feature.)

Thanks in advance.

**Comment by cscheid** [comment]
It's currently not possible, and won't be for the short-to-medium time frame. `freeze` refers to the "execution engines" of knitr or jupyter. `mermaid` cells, on the other hand, are evaluated in TypeScript on a subsequent part of the process.
